### ISSUE TITLE
Cyclospora cayetanensis in stool smear

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#11648]
**Preferred term label:**
Cyclospora cayetanensis in stool smear

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Cyclosporiasis

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
I am wondering if this should be
Positive stool Cyclospora cayetanensis oocyst test
Def: Detection of Cyclospora cayetanensis oocysts in a stool specimen by microscopy.
COmment: Cyclospora cayetanensis is a coccidian protozoan parasite of the human small intestine. The transmission stage of Cyclospora, known as an oocyst, is spherical and 8 to 10 μm in diameter. Unsporulated oocysts are shed into the environment with the feces of infected individuals. Oocysts undergo sporulation, becoming infective within a few days or weeks, depending upon environmental conditions. Infection may result when sporulated oocysts are ingested, primarily through contaminated water or food. Diagnosis usually involves microscopic examination of stool specimens for the presence of oocysts. 
PMID:15872269

Or does the term need to include other life stages?

**Comment by MickeySegal** [comment]
That checks out as correct, though https://uts.nlm.nih.gov/uts/umls/concept/C2020462 was more vague.  

BTW, typo "COmment".

**Comment by pnrobinson** [comment]
@MickeySegal 
Added with above definition as HP:0020358
